src/components/Touchable.tsx

```diff
@@ -1,15 +1,27 @@
 import * as React from 'react'
-import { TouchableWithoutFeedbackProps } from 'react-native'
+import { TouchableWithoutFeedbackProps, View } from 'react-native'
 import PlatformTouchable from 'react-native-platform-touchable'
 
 export interface Props extends TouchableWithoutFeedbackProps {
   borderless?: boolean
   children: React.ReactNode // must only have one direct child. see https://github.com/react-native-community/react-native-platform-touchable#touchable
+  borderRadius?: number
 }
 
-export default function Touchable({ borderless, ...passThroughProps }: Props) {
+/**
+ * @param borderless - If true, the touchable has a borderless ripple effect.
+ * @param borderRadius - Adds a border radius to the touchable and a parent view to prevent Android ripple overflow.
```

Comment:
```suggestion
 * @param borderRadius - should be added if the touchable component has rounded corners, to prevent the ripple effect from going outside the component on tap (Android)
```